website/versioned_docs/version-1.0.0/contracts/cargo-contract.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Cargo Contract: The ink! CLI
```

Comment:
The versioned doc has removed the quotes, it caused the related translation doc to be:
```
---
title:
  Cargo Contract: The ink! CLI
id: version-1.0.0-cargo-contract
original_id: cargo-contract
---
```

Reply:
@JoshOrndorff and I have a discussion, and we decided to add the quotes to solve it.

Reply:
I now realize that cutting a version is dropping the quotes from titles in _all_ cases. We only notice it in the ones that have colons (`:`). Maybe a better solution is to not put colons in the title?